src/main/kotlin/in/rcard/assertj/arrowcore/errors/NonEmptyListContains.kt

```diff
@@ -0,0 +1,27 @@
+package `in`.rcard.assertj.arrowcore.errors
+
+import arrow.core.NonEmptyList
+import org.assertj.core.error.BasicErrorMessageFactory
+
+/**
+ * Build error message when a [NonEmptyList] contains a specific value.
+ *
+ * @author Hamza Faraji
+ * @since 1.2.0
+ */
+class NonEmptyListContains(message: String, actual: NonEmptyList<Any?>, expected: Any?) :
+    BasicErrorMessageFactory(message, actual, expected)  {
+    companion object {
+        private const val EXPECTING_NOT_TO_CONTAIN =
```

Comment:
Yes, I thought it would be better to name errors in a affirmative way so that `NonEmptyListContains` is an actual error and that the message explains what was expected, I changed it to follow the existing convention like `EitherShouldBeLeft`.`shouldBeLeft` but it still reads like an assertion :grin: